ruby_02-web_applications_with_ruby/outlines/model_testing_in_sinatra.markdown

```diff
@@ -69,114 +69,162 @@ run TaskManagerApp
 
 Remove any old configuration settings inside of your controller.
 
-Remove any `require` or `require_relative` statements in your controller and/or models except for `require 'yaml/store'` in TaskManager. Our `environment.rb` file now requires all of our controllers, models, and views for us.
+Remove any `require` or `require_relative` statements in your controller and/or models except for `require 'sqlite3'` in Task. Our `environment.rb` file now requires all of our controllers, models, and views for us.
 
 ### Setting up Model Tests
 
-Add `gem 'minitest'` to your Gemfile and then bundle.
+Add the following lines to your `Gemfile`
+
+```ruby
+gem 'rack-test'
```

Comment:
So we probably don't need this as well if we aren't using rack-test anymore 👍 I'm going to go ahead and merge this though and make the update on master!